sites/public/cypress/integration/eligibility-questionnaire.ts

```diff
@@ -0,0 +1,69 @@
+describe("Verifying the eligibility questionnaire flow", () => {
+  it("Clicks the button on the homepage to launch the eligibility questionnaire", () => {
+    cy.visit("/")
+
+    // Find and click the button that says "Check My Eligibility"
+    const checkEligibilityButton = cy.contains("Check My Eligibility")
+
+    // Click the eligibility button and verify it takes us to the questionnaire welcome page
+    checkEligibilityButton.click()
+    cy.url().should("include", "/eligibility/welcome")
+  })
+
+  it("Navigates through the eligibility questionnaire flow", () => {
+    cy.visit("/eligibility/welcome")
+
+    // Verify that the page welcomes us :)
+    cy.contains("Welcome")
```

Comment:
This feels a little brittle to check that strings exist on the page. I'm thinking that if we even just change the strings that we use for each page it will fail. Maybe it's enough to ensure that the URL is at the correct page?

Reply:
Yeah sounds good - I got rid of all of the checks for specific text, but kept the URL checks.